### PR TITLE
feat: add integration test to init tunnel func

### PR DESCRIPTION
**Pull request by ternakkode**
This merge request adds a unit test for the InitTunnel function in the tunnel_test.go file. The test ensures that the function initializes the tunnel correctly and returns the expected response.

### CODE REVIEW

**Review by huzaifamk — CHANGES_REQUESTED** [review]
Good work @ternakkode , can you re-open this PR into `development` branch instead of `main`? This can be merged then.

File: server/handlers/tunnel.go [hunk 1]
```diff
@@ -19,17 +19,15 @@ func InitTunnel(w http.ResponseWriter, r *http.Request) {
 	fmt.Println("\n\n*************")
 	fmt.Println(r.Method) // > GET  | > POST
 	fmt.Println(r.URL)    // (http://localhost:5000/api/v1 ) > /api/v1
-	params := r.URL.Query()
-	var backend string
-	if _, ok := params["backend"]; !ok {
+
+	backend := r.URL.Query().Get("backend")
```
Comment (NJavokhir) [inline server/handlers/tunnel.go:23]: Good way to refactor code. I liked it.

File: server/handlers/tunnel_test.go
Comment (NJavokhir) [inline server/handlers/tunnel_test.go:1]: Overall, the test code appears well-written, thorough, and adheres to best practices.

**Review by NJavokhir — COMMENTED** [review]
Overall, the test code appears well-written, thorough, and adheres to best practices.

**Review by huzaifamk — APPROVED** [review]

**Pull request merged**